src/redis-cli.c

```diff
@@ -239,6 +239,7 @@ static struct config {
     int output; /* output mode, see OUTPUT_* defines */
     int push_output; /* Should we display spontaneous PUSH replies */
     sds mb_delim;
+    sds raw_delim;
```

Comment:
i would call it `cmd_delim`, i.e. this var and the one above it both target raw format, once used between mult-bulk, and one between commands